create-hpc-cluster/linuxnode-rdma-disabled.json

```diff
@@ -194,7 +192,9 @@
             "settings": {
               "fileUris": [
                 "[concat(parameters('scriptBaseUri'), 'updatednsrecord.sh')]"
-              ],
+              ]
+            },
+            "protectedSettings": {
```

Comment:
I think we're fine with leaving this outside of protectedSettings since it doesn't pass in a password